.gitlab-ci.yml

```diff
@@ -13,12 +13,11 @@ stages:
   - kubernetes
   - flaming-fir
 
-image:                             parity/rust-builder:latest
-
 variables:
   GIT_STRATEGY:                    fetch
   CARGO_HOME:                      "/ci-cache/${CI_PROJECT_NAME}/cargo/${CI_JOB_NAME}"
   SCCACHE_DIR:                     "/ci-cache/${CI_PROJECT_NAME}/sccache"
+  CARGO_INCREMENTAL:               0
```

Comment:
Why do you disable incremental?

Reply:
It doesn't make sense:
- we do not save and share the `target` folder between the pipelines/jobs
- `sccache` avoids everything incremental
- empirically I found that `sccache`ing while `CARGO_INCREMENTAL=0` is faster

I guess, nothing should go wrong with that.